main/src/routes/spotifyRouter.js

```diff
@@ -43,57 +31,20 @@ router.get("/", function (req, res) {
  * Handles GET request for spotify/login page
  */
 router.get("/login", (req, res) => {
-    let main = req.app.locals.main;
-
-    if (!main || !main.spotifyApi) {
-        req.app.locals.main = new Main();
-        main = req.app.locals.main;
-    }
-
-    const spotifyApi = main.spotifyApi;
     const html = spotifyApi.createAuthorizeURL(scopes);
-    console.log(html);
-
-    const state = generateRandomString(16);
-    res.cookie(stateKey, state);
-    res.redirect(html + "&state=" + state + "&show_dialog=true");
```

Comment:
To clarify, we don't need this anymore because we're using MemoryStore?

Reply:
Yup! I also rmbr reading that `express-session` might have bugs when used alongside `cookies`

Reply:
Oooh, interesting!! Is it safe for us to remove it? Cause I remember you mentioned this was like important for security?

Reply:
I think it should be okay. Because if someone wants to steal the access token data, they need to correctly guess the session ID AND our secret.